src/main/kotlin/no/nav/familie/ef/sak/service/PersonopplysningerService.kt

```diff
@@ -0,0 +1,72 @@
+package no.nav.familie.ef.sak.service
+
+import no.nav.familie.ef.sak.api.dto.*
+import no.nav.familie.ef.sak.integration.dto.pdl.KontaktadresseType
+import no.nav.familie.ef.sak.integration.dto.pdl.MotpartsRolle
+import no.nav.familie.ef.sak.integration.dto.pdl.PdlSøker
+import no.nav.familie.ef.sak.integration.dto.pdl.tilFormatertAdresse
+import org.springframework.stereotype.Service
+import java.time.LocalDate
+
+@Service
+class PersonopplysningerService(private val personService: PersonService) {
+
+    fun hentPdlSøker(ident: String): PersonopplysningerDto {
+        val søker = personService.hentPdlPerson(ident)
+        val fullmakter = søker.fullmakt.filter { it.motpartsRolle == MotpartsRolle.FULLMEKTIG }
+        val sivilstand = søker.sivilstand
+        //val andreIdenter = fullmakter.map { it.motpartsPersonident } + sivilstand.map { it.relatertVedSivilstand }.filterNotNull() //TODO hent personinfo
+        return PersonopplysningerDto(
+                statsborgerskap = søker.statsborgerskap.map {
+                    StatsborgerskapDto(land = it.land,
+                                       gyldigFraOgMed = it.gyldigFraOgMed,
+                                       gyldigTilOgMed = it.gyldigTilOgMed)
+                },
+                folkeregisterpersonstatus = Folkeregisterpersonstatus.fraPdl(søker.folkeregisterpersonstatus.single()),
+                sivilstand = sivilstand.map {
+                    SivilstandDto(type = Sivilstandstype.valueOf(it.type.name),
+                                  gyldigFraOgMed = it.gyldigFraOgMed,
+                                  relatertVedSivilstand = it.relatertVedSivilstand,
+                                  navn = null) //TODO
+                },
+                telefonnummer = søker.telefonnummer.find { it.prioritet == 1 }
+                        ?.let { TelefonnummerDto(it.landskode, it.nummer) },
```

Comment:
landskode -> landkode

Reply:
Dette er en del av Pdl domeneobjekt, og den bruker landskode. Jag kan bruke @JsonProperty for å overridea den